src/Sulu/Bundle/MediaBundle/Resources/config/services.xml

```diff
@@ -292,7 +299,7 @@
             <argument type="service" id="doctrine.orm.entity_manager" />
             <argument type="service" id="security.token_storage" on-invalid="null"/>
             <argument type="string">%sulu_media.collection.previews.format%</argument>
-            <argument>%sulu_security.permissions%</argument>
+            <argument type="string">%sulu_security.permissions%</argument>
```

Comment:
Type string is not correct here also if it seems to have no effect I would remove it:

```suggestion
            <argument>%sulu_security.permissions%</argument>
```